chat/src/main/java/com/github/twitch4j/chat/events/channel/RaidEvent.java

```diff
@@ -0,0 +1,41 @@
+package com.github.twitch4j.chat.events.channel;
+
+import com.github.twitch4j.chat.events.AbstractChannelEvent;
+import com.github.twitch4j.common.events.domain.EventChannel;
+import com.github.twitch4j.common.events.domain.EventUser;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Value;
+
+/**
+ * This event gets called when a user receives a raid.
+ * <p>
+ * This event is <i>not</i> called when a user receives a host that is not part of a raid.
+ */
+@Value
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class RaidEvent extends AbstractChannelEvent {
```

Comment:
`@Data` is enough to implement aboved annotations

Reply:
I used the code in the other `com.github.twitch4j.chat.events.channel` events as a model.

Reply:
`@Value` is the immutable version of `@Data` without setters obviously, it's perfectly fine like this.

Reply:
I see